CMakeLists.txt

```diff
@@ -2030,9 +2030,9 @@ function(curl_transform_makefile_inc _input_file _output_file)
   string(REPLACE "$(top_srcdir)"   "\${PROJECT_SOURCE_DIR}" _makefile_inc_text ${_makefile_inc_text})
   string(REPLACE "$(top_builddir)" "\${PROJECT_BINARY_DIR}" _makefile_inc_text ${_makefile_inc_text})
 
-  string(REGEX REPLACE "\\\\\n" "!π!α!" _makefile_inc_text ${_makefile_inc_text})
+  string(REGEX REPLACE "\\\\\n" "!^!^!" _makefile_inc_text ${_makefile_inc_text})
```

Comment:
What is this used for? Is there a significance for the characters?

Reply:
I think the idea is to use something that cannot occur in the original
`Makefile.inc`, nor in the generated `Makefile.inc.cmake`. Anything
fitting that would do. Using Unicode here was perhaps overkill.
Perhaps `!^!^!` still is!

Another option is to strip the newlines without reinserting them after
transformation. With the slight downside of making these interim files
less human-readable due to the long lines (libssh2 does it like that):

```cmake
  string(REGEX REPLACE "\\\\\n" "" _makefile_inc_text ${_makefile_inc_text})
  string(REGEX REPLACE "([a-zA-Z_][a-zA-Z0-9_]*)[\t ]*=[\t ]*([^\n]*)" "set(\\1 \\2)" _makefile_inc_text ${_makefile_inc_text})
```